src/Focusable.ts

```diff
@@ -139,16 +139,17 @@ export class FocusableAPI implements Types.FocusableAPI {
     }
 
     findNext(options: Types.FindNextProps): HTMLElement | null | undefined {
-        return this.findElement({
-            ...options,
-        });
+        const opts = { ...options };
+        const ret = this.findElement(opts);
+        options.outOfDOMOrderResult = opts.outOfDOMOrderResult;
+        return ret;
     }
 
     findPrev(options: Types.FindNextProps): HTMLElement | null | undefined {
-        return this.findElement({
-            isBackward: true,
-            ...options,
-        });
+        const opts = { ...options, isBackward: true };
+        const ret = this.findElement(opts);
+        options.outOfDOMOrderResult = opts.outOfDOMOrderResult;
+        return ret;
```

Comment:
Not sure I'm following what's happening here, seems like `findElement` is doing way more than just finding elements?!

Reply:
Why? It finds elements and returns metadata about the found elements.